src/main/java/com/maybe/maybe/entity/AbstractEntity.java

```diff
@@ -13,7 +13,7 @@
 @MappedSuperclass
 public abstract class AbstractEntity implements Serializable {
     @Id
-    @GeneratedValue(strategy = GenerationType.TABLE)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
```

Comment:
Please, don't include this file AbstractEntity in your pull request because I've already changed it so.
Otherwise, add
@Version
private Long version;
and it's getters and setters